PatchesAndAreasApi.Tests/Dockerfile

```diff
@@ -13,27 +13,28 @@ ENV SONAR_TOKEN=$SONAR_TOKEN
 WORKDIR /app
 
 # Install and run sonar cloud scanner
-RUN apt-get update && apt-get install -y openjdk-11-jdk
-RUN dotnet tool install --global dotnet-sonarscanner --version 5.6.0
+RUN apt-get update && apt-get install -y openjdk-17-jdk && apt-get clean \
+&& dotnet tool install --global dotnet-sonarscanner
 ENV PATH="$PATH:/root/.dotnet/tools"
 
-RUN dotnet sonarscanner begin /k:"LBHackney-IT_patches-and-areas-api" /o:"lbhackney-it" /d:sonar.host.url=https://sonarcloud.io /d:sonar.login="${SONAR_TOKEN}"
+RUN dotnet sonarscanner begin /k:"LBHackney-IT_patches-and-areas-api" \
+/o:"lbhackney-it" /d:sonar.host.url=https://sonarcloud.io /d:sonar.login="${SONAR_TOKEN}"
 
 # Copy csproj and nuget config and restore as distinct layers
 COPY ./PatchesAndAreasApi.sln ./
 COPY ./PatchesAndAreasApi/PatchesAndAreasApi.csproj ./PatchesAndAreasApi/
 COPY ./PatchesAndAreasApi.Tests/PatchesAndAreasApi.Tests.csproj ./PatchesAndAreasApi.Tests/
 COPY /NuGet.Config /root/.nuget/NuGet/NuGet.Config
 
-RUN dotnet restore ./PatchesAndAreasApi/PatchesAndAreasApi.csproj
-RUN dotnet restore ./PatchesAndAreasApi.Tests/PatchesAndAreasApi.Tests.csproj
+RUN dotnet restore ./PatchesAndAreasApi/PatchesAndAreasApi.csproj \
+&& dotnet restore ./PatchesAndAreasApi.Tests/PatchesAndAreasApi.Tests.csproj
 
 # Copy everything else and build
 COPY . .
 
-RUN dotnet build -c Release -o out PatchesAndAreasApi/PatchesAndAreasApi.csproj
-RUN dotnet build -c debug -o out PatchesAndAreasApi.Tests/PatchesAndAreasApi.Tests.csproj
+RUN dotnet build -c Release -o out PatchesAndAreasApi/PatchesAndAreasApi.csproj \
+&& dotnet build -c debug -o out PatchesAndAreasApi.Tests/PatchesAndAreasApi.Tests.csproj
 
-CMD dotnet test
+CMD ["dotnet", "test"]
```

Comment:
Interesting. Will the RUN step get run after the CMD?

Reply:
Which probably doesn't matter much right now. As no code coverage is being collected anyway & the CCI orb should override the analysis done within the dockerfile as well.